internal/util/profile.go

```diff
@@ -15,13 +13,3 @@ func GetMemStats() string {
 	runtime.ReadMemStats(&memstats)
 	return fmt.Sprintf("Alloc: %s, Sys: %s, GC: %d, PauseTotalNs: %dns", humanize.Bytes(memstats.Alloc), humanize.Bytes(memstats.Sys), memstats.NumGC, memstats.PauseTotalNs)
 }
-
-func Tic(s string) time.Time {
-	log.Println("START:", s)
-	return time.Now()
-}
-
-func Toc(start time.Time) {
-	end := time.Now()
-	log.Println("END: ElapsedTime in seconds:", end.Sub(start))
-}
```

Comment:
These may be actually occasionally useful for profiling?

BTW I didn't even know about the existence of this `internal/util/profile.go` until now (and I find it rather nice). (And BTW also I didn't know about the existence of the `memusage` command.)

Reply:
It's not a big deal to write two lines when we need to profile something. For now, I propose removing redundant code that is not used.

Reply:
What exactly would we achieve by removing it?

Reply:
Removing unnecessary functions will allow refactoring the `util` package into something more meaningful with a better name.

The package name `util` is considered a bad package name according to https://go.dev/blog/package-names#bad-package-names.

Reply:
> refactoring the `util` package into something more meaningful with a better name.

Like what?

Reply:
> The package name `util` is considered a bad package name according to https://go.dev/blog/package-names#bad-package-names.

So it's the usual ideal world example which would have been nice in the moment it has been respected from the beginning. But unfortunately the world isn't ideal, it hasn't been considered from the beginning and thus `util` is grown historically by throwing everything into it which doesn't fit into the other base packages.
I wouldn't say that we're against such a cleanup process, but currently we've we have bigger construction sites at other places.

> Like what?

As said by @dmaluka or at least how I interpret it:
If you have a much better suggestion, please go ahead.
But remember, especially with the `lua.go` in `util` we most probably face problems with `plugin` interfaces.

Reply:
Reverted back `Tic/Toc`